### ISSUE TITLE
Reverse tab order in topics with tabs (2.x first)

### ISSUE THREAD

**Issue by guardrex** [aspnet/Docs#4089]
There are a number of topics with the tab order 1.x-2.x, but I understand it was settled that the topics should render the tab with the latest release info, thus they should be 2.x first. I don't have a list. I patched one and then realized it wasn't a one-off when I saw others that need the same work. Seems that all of them should be addressed on one PR.
